README.md

```diff
@@ -39,33 +39,34 @@ Deep learning practitioners should use one of the
 
 # Documentation
 
-* [Developer Guide] explains the programming model, supported functionality,
+* [Release Notes] explains the new features added, issues resolved, and improvements implemented in each version of oneDNN.
+* [oneDNN Developer Guide and Reference] explains the programming model, supported functionality,
   and implementation details, and includes annotated examples.
 * [API Reference] provides a comprehensive reference of the library API.
 
 [Developer Guide]: https://oneapi-src.github.io/oneDNN
 [API Reference]: https://oneapi-src.github.io/oneDNN/group_dnnl_api.html
+[Release Notes]: https://github.com/oneapi-src/oneDNN/releases/tag/v3.7
```

Comment:
This README.md lives on the main branch and is not specific for a particular release. Link to releases section is more appropriate here than the link to oneDNN v3.7 release notes.

```suggestion
[Release Notes]: https://github.com/oneapi-src/oneDNN/releases```